extra/math/text/english/english.factor

```diff
@@ -112,3 +112,17 @@ M: complex number>text
         [ 0 < " minus " " plus " ? ]
         [ abs number>text " i" append ] bi
     ] bi* 3append ;
+
+: ordinal-suffix ( n -- suffix )
```

Comment:
I had this floating around in my utils somewhere:

```factor
: humanize ( n -- str )
    dup 100 mod 11 13 between? [ "th" ] [
        dup 10 mod {
            { 1 [ "st" ] }
            { 2 [ "nd" ] }
            { 3 [ "rd" ] }
            [ drop "th" ]
        } case
    ] if [ number>string ] [ append ] bi* ;
```

It's weird to convert the number to a string then back to a number?

Reply:
Hmm... your implementation outputs `th` for `-101` rather than `st`, to me it's "negative one hundred first"

Reply:
Yeah, I didn't build it for negative numbers.  Was just pointing out using math only instead of strings.